### PR TITLE
add maintainers.md

### LINKED ISSUE #7: Add MAINTAINERS file (or CODEOWNERS)
It would be nice to have a `MAINTAINERS.md` file for whomever that wants to contact maintainers (or ask for feedback on PR, issues, etc) have a place to look at it; and also for keep track of how governance change over time.

I found also an interesting feature in GitHub, the ["code owners"](https://blog.github.com/2017-07-06-introducing-code-owners/), which automatically request review for maintainers if specify in a `CODEOWNERS` file (in the root directory or `.github/`). I think that is worth to look at it, and keep in the `.github/` folder this file with the templates for issues/PRs. More info here: https://blog.github.com/2017-07-06-introducing-code-owners/

What do you thing @lbergelson, @jacarey, @tfenne?

### PR DESCRIPTION
**Pull request by lbergelson**
I've added a MAINTAINERS.md file.  It's similar but to but will eventually be distinct from the authors file.

@samtools/htsjdk-next-maintainers any suggestions?

closes #7

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#### These are the current Htsjdk-Next-Beta project maintainers (sorted alphabetically by handle)
+
+-----
+
+* Jay Carey	**@jacarey** 	_jcarey@broadinstitute.org_
+*	Louis Bergelson	 **@lbergelson**	_louisb@broadinstitute.org_
```
Comment (magicDGS) [inline MAINTAINERS.md]: Maybe a markdown table instead? The spacing looks funny, and with a table even in plain-text could be perfectly visualized...

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#### These are the current Htsjdk-Next-Beta project maintainers (sorted alphabetically by handle)
+
+-----
+
+* Jay Carey	**@jacarey** 	_jcarey@broadinstitute.org_
```
Comment (magicDGS) [inline MAINTAINERS.md]: Maybe links to the github accounts might be nice to quickly jump to them

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+*	Louis Bergelson	 **@lbergelson**	_louisb@broadinstitute.org_
+* Daniel Gómez Sánchez **@magicDGS** 	_daniel.gomez.sanchez@hotmail.es_
```
Comment (magicDGS) [inline MAINTAINERS.md]: Using `<>` around the email is more explicit for indicating that it is an email without rendering...

**Comment by lbergelson** [comment]
@magicDGS I've responded to your comments.

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+-----
+
+
+| Github ID                                    | Name                   | Email                              |
+|----------------------------------------------|------------------------|------------------------------------|
+| [@jacarey](https://github.com/jacarey)       | Jay Carey              | <jcarey@broadinstitute.org>        |
+| [@lbergelson](https://github.com/lbergelson) | Louis Bergelson        | <louisb@broadinstitute.org>        |
+| [@magicDGS](https://github.com/magicDGS)     | Daniel Gómez Sánchez   | <daniel.gomez.sanchez@hotmail.es>  |
+| [@tfenne](https://github.com/tfenne)         | Tim Fennel             | <tfenne@tfenne.com>                |
```
Comment (tfenne) [inline MAINTAINERS.md]: Fennel -> Fennell please

Comment (lbergelson) [reply MAINTAINERS.md]: Ack, sorry!  
 
image

**Comment by lbergelson** [comment]
closes #7

**Pull request merged**